AzureCalling/External/Authentication/AADAuthHandler.swift

```diff
@@ -53,18 +65,23 @@ class AADAuthHandler {
     }
 
     // MARK: Public API
+    func login(presentVc: UIViewController,
```

Comment:
I'm going to simplify the login calls to this one method, so we can get interactive if necessary, otherwise non-interactive.